### PR TITLE
Add Note to clarify that is_site_enabled can only be called for one site

### LINKED ISSUE #1421: nidigital is_site_enabled documentation is misleading
### Description of issue

The [documentation for the nidigital is_site_enabled method](https://nimi-python.readthedocs.io/en/master/nidigital/class.html#is-site-enabled) includes the generic repeated capabilities tip.  Although nothing in the documentation is untrue, calling the method on more than one site at a time will result in an error, just like in the C API.  However, that is not mentioned in the documentation and the repeated capabilities tip makes it sound like calling the method on multiple sites should work.

### PR DESCRIPTION
**Pull request by phindman**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [ ] ~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~

- [ ] ~I've added tests applicable for this pull request~

### What does this Pull Request accomplish?

Adds Note to the documentation for is_site_enabled, clarifying that it is an error to specify more than one site.

### List issues fixed by this Pull Request below, if any.

* Fix #1421

### CODE REVIEW

**Comment by marcoskirsch** [comment]
ok to test

**Review by marcoskirsch — APPROVED** [review]

File: src/nidigital/metadata/functions.py [hunk 1]
```diff
@@ -2080,6 +2080,7 @@
     'IsSiteEnabled': {
         'documentation': {
             'description': 'TBD'
+            'note': 'The function returns an error if more than one site is specified.\n'
```
Comment (sbethur) [inline src/nidigital/metadata/functions.py:2083]: Please include the updated file in `generated` folder in the review.

**Review by sbethur — CHANGES_REQUESTED** [review]

File: src/nidigital/metadata/functions.py (hunk 1 above)
Comment (sbethur) [inline src/nidigital/metadata/functions.py:2083]: Remember to submit this change in the internal metadata too.

Comment (phindman) [reply src/nidigital/metadata/functions.py:2083]: I rebased onto master and included the generated files.

**Review by sbethur — APPROVED** [review]

**Pull request merged**